renderdoccmd/3rdparty/cmdline/cmdline.h

```diff
@@ -788,11 +788,11 @@ class parser{
     }
 
   protected:
-    std::string full_description(const std::string &desc){
+   std::string full_description(const std::string &description){
       std::string defval = detail::default_value<T>(def);
 
       return
-        desc+" ("+
+        description+" ("+
```

Comment:
Generally I avoid making changes to third party files unless it is strictly necessary for functionality. An issue like this if it's a problem should be solved by adding warning suppressions at build time.